site/en/blog/new-in-chrome-104/index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: New in Chrome 104
+description: >
+  Chrome 104 is rolling out now! With region capture, you can now specify a
+  crop area when using getDisplayMedia to capture the current tab. Media
+  query syntax can be written using mathematical comparison operators.
+  Shared Element Transitions start a new origin trial. And there's plenty
+  more.
+layout: 'layouts/blog-post.njk'
+date: 2022-08-02
+authors:
+  - petelepage
+hero: 'image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/2lpDeA6BU9K7FdETYCrV.png'
+alt: >
+  New in Chrome hero logo
+tags:
+  - new-in-chrome
+  - chrome-104
+---
+
+{% YouTube id='GIt-g0nkt8w' %}
+
+Here's what you need to know:
+
+* You can now specify a crop area using [region capture](#region-capture)
+  when using `getDisplayMedia` to capture the current tab.
+* Media query syntax can be written using
+  [mathematical comparison operators](#mq-math).
+* [Shared Element Transitions](#shared-element-transitions) start a new
+  origin trial.
+* And there's plenty [more](#more).
+
+I'm [Pete LePage](https://petelepage.com). Let's dive in and
+see what's new for developers in Chrome 104.
+
+## Specify a crop area with region capture {: #region-capture }
+
+`getDisplayMedia` makes it possible to create a video stream from the current
+tab. But, there are times when you don’t want the entire tab, only a small
+portion of it. Until now, the only way to do that was to manually crop each
+video frame.
+
+With Region Capture, a web app can define the specific area of the screen it
+wants to share. For example, Google Slides could allow you to stay in the
+standard editing view, and share the current slide.
+
+<figure>
+  {% Img src="image/vvhSqZboQoZZN9wBvoXq72wzGAf1/L87rANm3IfsVZiMHfbH2.png", alt="Screenshot of a browser window featuring a web app highlighting the main content area and cross-origin iframe.", width="800", height="568" %}
+  <figcaption>
+    The main content area is in blue and the cross-origin iframe is in red.
+  </figcaption>
+</figure>
+
+To use it, select the element to share, then create a new `CropTarget` based on
+that element. Next, start screen sharing by calling `getDisplayMedia()`, that
```

Comment:
```suggestion
that element. Next, start screen sharing by calling `getDisplayMedia()`. That
```

Reply:
I think you were intending a subordinate clause to modify `getDisplayMedia()`. The correct conjunction should have been 'which', but without a comma. This suggestions fixes the grammar without changing the word.